scalableminds/webknossos
# Allow to disable automatic reloading of meshes during proofreading

### Steps to test:
- See prerequisites in issue #6976 
- Activate proofreading mode in toolbar
- If switch button is activated, meshes should render as before when clicking a segment
- If it is deactivated, meshes should not be rendered after clicking a segment
- Setting should be stored for a user, so logging on and off shouldn't change the setting of the switch

### Issue:
- fixes #6976

------
